src/System.Net.Primitives/tests/UnitTests/System.Net.Primitives.Unit.Tests.projinc

```diff
@@ -104,10 +122,48 @@
     <Compile Include="$(CommonPath)\Interop\Windows\NtDll\Interop.RtlIpv6StringToAddressEx.cs" >
       <Link>ProductionCode\Common\Interop\Windows\NtDll\Interop.RtlIpv6StringToAddressEx.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\Winsock\Interop.ErrorCodes.cs" >
-      <Link>ProductionCode\Common\Interop\Windows\Winsock\Interop.ErrorCodes.cs</Link>
+  </ItemGroup>
+  
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
```

Comment:
@pgavlin Given that UTs must be "pure" and make no system calls, the following files need to be removed and fakes added instead.